Src/java/elm/src/main/java/org/cqframework/cql/elm/visiting/ElmBaseLibraryVisitor.java

```diff
@@ -71,4 +99,19 @@ public T visitIncludeDef(IncludeDef elm, C context) {
     public T visitContextDef(ContextDef elm, C context) {
         return null;
     }
+
+    public T visitStatements(Statements elm, C context) {
+        int n = elm.getDef().size();
+        for (int i=0; i<n; i++) {
+            Element c = elm.getDef().get(i);
+            if (c instanceof ExpressionDef) {
+                visitElement((ExpressionDef)c, context);
+            } else if (c instanceof ContextDef) {
+                visitElement((ContextDef)c, context);
+            } else if (c instanceof FunctionDef) {
+                visitElement((FunctionDef)c, context);
```

Comment:
FunctionDef is a subclass of of ExpressionDef, so it needs to be first in the list of ifs. Otherwise this will never get hit.